### PR TITLE
Seller Model

### PR DESCRIPTION
**Pull request by LoveHope5**
Here is the seller model file

### CODE REVIEW

**Review by Ajioz — COMMENTED** [review]
Hi there,

Nice work going around.

I am sure you intend to make a PR to your session lead repo.

According to the workflow, the team lead would make a PR from his repo to the Gomerce repo where we'll review all the work your team has done. 

**Please note:**
Your team lead would be willing to review your PR against his/her repo and merge on that level.  Meaning he shall do this for every member of his team, and make a single PR to the Gomerce repo, which is a sum of your teamwork.

File: src/models/seller.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+"""
+from . import db
+from .abc import BaseModel, MetaBaseModel
+from datetime import datetime
+
+from werkzeug.security import generate_password_hash, check_password_hash
+
+
+class Seller(db.Model, BaseModel, metaclass=MetaBaseModel):
+    """ The Customer model """
```
Comment (samdoghor) [inline src/models/seller.py:12]: Seller not Customer

File: src/models/seller.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+    __tablename__ = "sellers"
+
+    id = db.Column(db.Integer, primary_key=True)
+    store_id = db.Column(db.Integer)
```
Comment (samdoghor) [inline src/models/seller.py:17]: Store_id is a foreign key, you did indicate it in your file

**Review by samdoghor — CHANGES_REQUESTED** [review]
- Make your PR to samdoghor/GomerceB repo not the central repo.
- the following columns are missing: phone_verified and rating

**Comment by samdoghor** [comment]
Also, please git pull before making your new changes and resolve every conflicts

**Pull request closed without merging**